instance/models/openedx_appserver.py

```diff
@@ -241,12 +253,14 @@ def github_admin_username_list(self):
         """
         Returns the github usernames of this instance admins
 
-        Admins are the members of the default team of the `github_admin_organization_name` org
+        Admins are all users listed in github_admin_users and the members of the default team of the
+        organizations in github_admin_organizations.
         """
-        if self.github_admin_organization_name:
-            return get_username_list_from_team(self.github_admin_organization_name)
-        else:
-            return []
+        admin_users = []
```

Comment:
@haikuginger You are right with that.  I thought about it when I wrote the code, and decided that it's good enough as it is.  The case is rare, and it works fine as it is, you will only end up with the SSH keys of the affected users added to `authorized_keys` twice, which doesn't really hurt.